python/pyspark/sql/connect/window.py

```diff
@@ -242,3 +243,46 @@ def rangeBetween(start: int, end: int) -> "WindowSpec":
 
 
 Window.__doc__ = PySparkWindow.__doc__
+
+
+def _test() -> None:
+    import os
+    import sys
+    import doctest
+    from pyspark.sql import SparkSession as PySparkSession
+    from pyspark.testing.connectutils import should_test_connect, connect_requirement_message
+
+    os.chdir(os.environ["SPARK_HOME"])
+
+    if should_test_connect:
+        import pyspark.sql.connect.window
+
+        globs = pyspark.sql.window.__dict__.copy()
+        # Works around to create a regular Spark session
+        sc = SparkContext("local[4]", "sql.connect.window tests", conf=SparkConf())
+        globs["_spark"] = PySparkSession(sc, options={"spark.app.name": "sql.connect.window tests"})
+
+        # Creates a remote Spark session.
+        globs["spark"] = PySparkSession.builder.remote("sc://localhost").getOrCreate()
+
+        (failure_count, test_count) = doctest.testmod(
+            pyspark.sql.connect.window,
+            globs=globs,
+            optionflags=doctest.ELLIPSIS
+            | doctest.NORMALIZE_WHITESPACE
+            | doctest.IGNORE_EXCEPTION_DETAIL,
+        )
+        # TODO(SPARK-41529): Implement stop in RemoteSparkSession.
+        #   Stop the regular Spark session (server) too.
```

Comment:
```suggestion
        globs["spark"].stop()
```

It's implemented now.

Reply:
@HyukjinKwon addressed comment